### PR TITLE
578 content update

### LINKED ISSUE #578: content-update

### PR DESCRIPTION
**Pull request by Arfah**
#closes #578

### CODE REVIEW

File: data/graduates.yml [hunk 1]
```diff
@@ -23,9 +23,63 @@
...
+    - question: What made you want to start learning to code in the first place?
+      answer: I felt I reached a point in my last career where I wasn't learning much. I happen to be working with some great developers who guided me into coding. I really enjoyed the idea of making things from scratch.
+    - question: How did you come across Makers Academy and what made you decide to apply?
+      answer: I was aware of Makers Academy for a while and when I stared to think of it as a career. I researched and looked into bootcamps more.
+    - question: What happened during/after graduation and where are you job-wise?
+      answer: I started looking for jobs straight away. I got a few interviews through Makers Academy and after a few weeks I had two offers.
+    - question: What advice would you give to people thinking of changing career?
+      answer: It just not worth spending the rest of your life not doing something you enjoy.
+    - question: What do you say to people who say £8000 is expensive?
+      answer: It is a lot of money but if you end up getting a career change out of it and feeling more happier then it is worth the money.
```
Comment (nikeshashar) [inline data/graduates.yml]: remove more

File: data/graduates.yml [hunk 2]
```diff
@@ -23,9 +23,63 @@
...
+    - question: What as your biggest worry before joining Makers Academy and how do you feel about it now?
```
Comment (nikeshashar) [inline data/graduates.yml]: What was your biggest mistake

File: data/graduates.yml [hunk 3]
```diff
@@ -23,9 +23,63 @@
...
 #       answer:
 
+- first_name: Emma
+  last_name: Beynon
+  image: emmab.jpg
+  summary: Digital Marketing to Junior Developer at Government Digital Service
+  testimonial: "My Makers Academy experience was intense, stressful, fulfilling and fun!"
+  cohort_month: February
+  cohort_year: 2016
+  categories:
```
Comment (nikeshashar) [inline data/graduates.yml]: Do we want to add categories? Female?

File: data/graduates.yml [hunk 4]
```diff
@@ -23,9 +23,63 @@
...
+    - question: What did you find difficult during the course?
+      answer: I found balancing my desire to know everything immediately with my mental sanity difficult.
+    - question: What happened during/after graduation and where are you job-wise?
+      answer: I spend 6 weeks as an alumni helper and then really focused on the job hunt and got a job as a junior software developer at Surfdome.
+    - question: What advice would you give to people thinking of changing career?
+      answer: Do a lot of research and figure out the expectation of how long it will take to get a new job and make sure you have enough money to last. Changing careers has been the best thing to happen to me. It takes work but is incredible rewarding.
+    - question: What do you say to people who say £8000 is expensive?
+      answer: Judge for yourself if it's worth it. You normally get a staring salary of around £32k for a junior developer. Equivalent masters courses are about the same price and this is a higher selective course so for me it made sense and coming out of the course I found the content to be worth it.
```
Comment (nikeshashar) [inline data/graduates.yml]: Junior Developer

Comment (nikeshashar) [inline data/graduates.yml]: higher => highly

File: data/graduates.yml [hunk 5]
```diff
@@ -23,9 +23,63 @@
...
+    - question: What as your biggest worry before joining Makers Academy and how do you feel about it now?
```
Comment (nikeshashar) [inline data/graduates.yml]: what was

File: data/graduates.yml [hunk 6]
```diff
@@ -23,9 +23,63 @@
...
+      answer: That I would not be able to get a job and I now have one!
+      answer: It's cliché but life changing!
```
Comment (nikeshashar) [inline data/graduates.yml]: Two answers?

File: data/graduates.yml [hunk 7]
```diff
@@ -23,9 +23,65 @@
...
+  cohort_month: February
+  cohort_year: 2016
+  categories:
+    - female 
+    - career changer 
+  case_study:
+    - question: What were you doing before joining Makers Academy?
+      answer: I was working in Digital Marketing.
+    - question: What made you want to start learning to code in the first place?
+      answer: I felt I reached a point in my last career where I wasn't learning much. I happen to be working with some great developers who guided me into coding. I really enjoyed the idea of making things from scratch.
```
Comment (nikeshashar) [inline data/graduates.yml]: I *happened* to be working with...

File: data/graduates.yml [hunk 8]
```diff
@@ -23,9 +23,65 @@
...
+    - question: How did you come across Makers Academy and what made you decide to apply?
+      answer: I was aware of Makers Academy for a while and when I stared to think of it as a career. I researched and looked into bootcamps.
```
Comment (nikeshashar) [inline data/graduates.yml]: think of it? (should it be think of coding as a career?)

Comment (danldb) [inline data/graduates.yml]: should be *started to*

File: data/graduates.yml [hunk 9]
```diff
@@ -23,9 +23,65 @@
...
+  summary: English and Maths Tutor to Junior Developer at Surfdom
+  testimonial: "It's cliché but Makers Academy was life changing!"
+  cohort_month: January
+  cohort_year: 2016
+  categories: career changer
+  case_study:
+    - question: What were you doing before joining Makers Academy?
+      answer: I was an English and Maths tutor.
+    - question: What made you want to start learning to code in the first place?
+      answer: I wanted an intellectual challenging and rewarding career and my best friend told me I should do it.
```
Comment (nikeshashar) [inline data/graduates.yml]: I wanted an *intellectually*

File: data/graduates.yml [hunk 10]
```diff
@@ -23,9 +23,65 @@
...
+    - question: How did you come across Makers Academy and what made you decide to apply?
+      answer: I research online and I saw a lot of recommendations on reddit. I read a lot of the student reviews and felt that a short course was better for me than a conversion masters.
```
Comment (nikeshashar) [inline data/graduates.yml]: I *researched* online and I saw ...

File: data/graduates.yml [hunk 11]
```diff
@@ -23,9 +23,65 @@
...
+    - question: What did you find difficult during the course?
+      answer: I found balancing my desire to know everything immediately with my mental sanity difficult.
+    - question: What happened during/after graduation and where are you job-wise?
+      answer: I spend 6 weeks as an alumni helper and then really focused on the job hunt and got a job as a junior software developer at Surfdome.
```
Comment (nikeshashar) [inline data/graduates.yml]: I *spent* 6 weeks as an ...

File: data/graduates.yml [hunk 12]
```diff
@@ -23,9 +23,65 @@
...
+    - question: What advice would you give to people thinking of changing career?
+      answer: Do a lot of research and figure out the expectation of how long it will take to get a new job and make sure you have enough money to last. Changing careers has been the best thing to happen to me. It takes work but is incredible rewarding.
+    - question: What do you say to people who say £8000 is expensive?
+      answer: Judge for yourself if it's worth it. You normally get a staring salary of around £32k for a Junior Developer. Equivalent masters courses are about the same price and this is a highly selective course so for me it made sense and coming out of the course I found the content to be worth it.
+    - question: What as your biggest worry before joining Makers Academy and how do you feel about it now?
+      answer: That I would not be able to get a job and I now have one!
+      answer: It's cliché but life changing!
```
Comment (nikeshashar) [inline data/graduates.yml]: There's two answer blocks here

**Comment by pitchinvasion** [comment]
Ed is going to review this himself tonight and squirrel it (he just asked me if he could)

**Comment by Arfah** [comment]
Coolieo

**Comment by pitchinvasion** [comment]
OK Ed's changes are in, :shipit: if you updated stuff from nikesh's comments. @nikeshashar I'd normally recommend just making changes yourself it's the easiest way

**Pull request merged**